app/controllers/developer_controller.rb

```diff
@@ -13,7 +13,18 @@ def update_sites
 
   def production_log
     @log = if params[:grep].present?
-             `tail -n 10000 log/production.log | grep -E '#{params[:grep]}' -C #{params[:context]} --color=never`
+             begin
+               grep_str = params[:grep].instance_of?(String) ? params[:grep] : params[:grep].to_s
+               context_int = params[:context].instance_of?(Integer) ? params[:context] : params[:context].to_i
```

Comment:
You can just run `params[:grep].to_s`. `params[:grep]` will always be either nil or a string. Same is true for `params[:context]`.